op-challenger/metrics/metrics.go

```diff
@@ -58,6 +62,12 @@ func NewMetrics() *Metrics {
 			Name:      "up",
 			Help:      "1 if the op-challenger has finished starting up",
 		}),
+		cannonExecutionTime: factory.NewHistogram(prometheus.HistogramOpts{
+			Namespace: Namespace,
+			Name:      "cannon_execution_time",
+			Help:      "Time (in seconds) to execute cannon",
+			Buckets:   append([]float64{1.0, 10.0}, prometheus.LinearBuckets(30.0, 30.0, 14)...),
```

Comment:
If I understand this correctly it's only tracking up to 7 minutes but the initial run of cannon is expected to take between 1 and 2 hours. I suspect we want to use exponential buckets.

Reply:
Updated to use exponential buckets which should cover a few hours if i did my math right